app/Http/Controllers/ConfigurableText/ConfigurableTextController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers\ConfigurableText;
+
+use App\Models\User;
+use App\Models\ConfigurableText;
+use App\Models\Workshop;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Database\Eloquent\Casts\Attribute;
+
+class ConfigurableTextController extends Controller
+{
+    public const CUSTOM_TEXT_FIELDS = [
+        'APPLICANT_REGISTRATION',
+        'TENANT_REGISTRATION',
+        'APPLICATION_INFORMATION_PRIOR_TO_FINALIZATION',
+        'APPLICATION_FILES',
+        'APPLICATION_INFORMATION_AFTER_FINALIZATION',
+        'APPLICATION_QUESTION2',
+        'APPLICATION_QUESTION2_HELPER',
+        'APPLICATION_QUESTION3',
+        'APPLICATION_QUESTION3_HELPER',
+        'APPLICATION_QUESTION4',
+        'APPLICATION_QUESTION4_HELPER',
```

Comment:
Hm, I'm not sure I like this being _partially_ configurable (why is the number of questions fixed?).

I'd prefer if we either keep these hard-coded (I don't think we'd want to change these questions _during_ an ongoing application period) or go for the fully configurable approach (configurable number of questions). This design was strange to begin with (where did question 1 go?!)